### ISSUE TITLE
Support for fail2ban with abuseipdb.com

### ISSUE THREAD

**Issue by Dexus** [mailcow/mailcow-dockerized#5330]
### Summary

typical I run multiple server, on each I have a additional support to report abused IPs to abuseipdb.com.
Also I get this blacklist an ban them for a while. This can be automate the ban for already known IPs for abuse usage.

### Motivation

It will automatic report bad ips, and also we will get faster updates about abuse and can ban them befor they are trying it the first time.

**Comment by patschi** [comment]
There is a community-supported custom-made script to automatically add IPs from AbuseIPDB to mailcow's blocklist: https://docs.mailcow.email/third_party/abuseipdb/third_party-abuseipdb/

I think that would do the trick, IMHO making a native integration not much more beneficial. A more custom approach might allow more tight control of the bans and possible integration with other services.

**Comment by Dexus** [comment]
@patschi I don't think so. Becuase I don't have the way to AbuseIPDB only the way from AbuseIPDB... thats a complete different direction.

A normal and easier way to modificate the fail2ban would make many things better.
